### PR TITLE
Documentation for load and loads and updated README

### PR DESCRIPTION
**Pull request by pbvarga1**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
...
+pvl.load
++++++++++
+
+General:: 
+
+This module parses an isis label from a stream and returns a dictionary 
+containing information from the label. The stream must be a string of the image 
+file name with the path to the file included.
+
+ >>> from pvl import load
```
Comment (wtolson) [inline README.rst]: I think we should prefer the import syntax for docs as it reads bit more clearly. For example:
```python
>>> import pvl
>>> pvl.load(img)
```

File: docs/index.rst [hunk 2]
```diff
@@ -14,9 +14,12 @@ Contents:
    readme
    installation
    usage
+   load
```
Comment (wtolson) [inline docs/index.rst]: I would combine these two sections into one called parsing. Right now there's too much duplication as the only difference between `load` and `loads` is the input is a stream vs a string.

**Comment by wtolson** [comment]
This looks awesome! Thanks for getting this started. I left a few comments with my thoughts.

File: README.rst (hunk 1 above)
Comment (godber) [inline README.rst]: Yeah, since the module and function name are short, I think that makes sense and is the most explicit way to do things.

**Comment by wtolson** [comment]
Also this package was modeled after the python standard lib [json](https://docs.python.org/2/library/json.html) package. You might find its docs useful for both understanding and modeling our docs.

File: README.rst [hunk 3]
```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
...
+How to use Module
+--------------------
+
+.. contents:: `Table of Contents`
+	:local:
+
+pvl.load
++++++++++
+
+General::
```
Comment (godber) [inline README.rst]: Delete this line.

File: README.rst [hunk 4]
```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
...
+
+.. contents:: `Table of Contents`
+	:local:
+
+pvl.load
++++++++++
+
+General:: 
+
+This module parses an isis label from a stream and returns a dictionary
```
Comment (godber) [inline README.rst]: Change "This module parses an isis label from" to "This module parses an PVL compliant label from"

File: README.rst [hunk 5]
```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
```
Comment (godber) [inline README.rst]: We need to explain here what PVL is and what label types are compatible with it.  I have tried to read the specification here: http://public.ccsds.org/publications/archive/647x2b1.pdf to figure that out but its not jumping out at me.  I am not going to hold this PR up for this, but we should figure this out.  @wtolson do you know anything offhand?

File: README.rst (hunk 3 above)
Comment (pbvarga1) [inline README.rst]: Which line exactly?

File: README.rst (hunk 5 above)
Comment (wtolson) [inline README.rst]: Yeah, I was planing on doing some thinking about explaining what pvl is, where it came from and the approach we're taking in accepting the various "dialects".

File: README.rst [hunk 6]
```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
...
+ u'PANCAM_RIGHT'
+
+See full documentation :doc:`parsing`
+
+pvl.loads
++++++++++
+
+General::
+
+This module parses an isis label from a string and returns a dictionary
```
Comment (godber) [inline README.rst]: Same change for isis in this line too.

File: docs/load.rst [hunk 7]
```diff
@@ -0,0 +1,139 @@
+========
+pvl.load
+========
+
+.. contents:: `Table of Contents`
+  :local:
+
+This module parses an isis label from a stream and returns a dictionary
```
Comment (godber) [inline docs/load.rst]: Correct isis statement again

**Comment by godber** [comment]
Please find all of the cases where you say isis label when you mean PVL compatible label.  In cases when you're opening a PDS3 `.img` file you should say PDS3 label, in cases where you're opening an Isis cub file you can say Isis label.  Keep in mind, PVL parses multiple label types, Isis and PDS3 and more even ... you have to be careful about which one you're talking about or if you just talking about labels in general.

Please spell check, there are many spelling errors and typos.

**Pull request merged**